build/squire-raw.js

```diff
@@ -2093,6 +2093,7 @@ var findPreviousTextOrNotEditable = function(root, node){
 
 Squire.prototype.enter = function (self, event, range) {
   self  = self  ? self  : this
+  event.preventDefault()
```

Comment:
I committed to master the build of the latest commits, github has yet to correct this diff